src/main/groovy/org/labkey/gradle/plugin/JavaModule.groovy

```diff
@@ -241,14 +258,20 @@ class JavaModule extends FileModule
 
         config = labkeyConfig == null ? config : (config == null ? labkeyConfig : config + labkeyConfig)
 
-        // trim nothing from api
-        String apiProjectPath = BuildUtils.getProjectPath(project.gradle, "apiProjectPath", ":server:api")
-        if (project.path.equals(apiProjectPath))
+//        // trim nothing from api
+//        if (BuildUtils.isApi(project))
+//            return config
+//        else // all other modules should remove everything in the dedupe configuration
+//        {
+//            return config - project.configurations.dedupe
+//        }
```

Comment:
Commented out because it doesn't work as expected.  I posted to the Gradle forums about this but haven't heard anything.  I need to see if I can tease out a simple repro case so I can file a bug with the Gradle folks.